### ISSUE TITLE
Dark Theme Text Color Issue

### ISSUE THREAD

**Issue by python3xdev** [eranif/codelite#3028]
I recently started using codelite, I have only found one issue, and its with the dark theme. The theme I have selected is called " Atom One-Dark". The **Build Tab** has black text color and its hard to read. The **Output Tab** has white text color, as it should be. Is there something I can do to fix this myself or must I wait until an update? Here are some images displaying the problem:

## Build Tab (supposed to be white text):
![image](https://user-images.githubusercontent.com/77379766/175407395-f850da62-40cb-4b99-b7da-af3fda6364a2.png)

## Output Tab:
![image](https://user-images.githubusercontent.com/77379766/175407616-45112baf-c338-4fba-912f-ffd897111e51.png)

**Comment by eranif** [comment]
The build tab colours are selected as the colour as your main editor area.
which theme are you using?
also, when changing theme from: Settings -> colours and fonts, does it make a difference?

**Comment by python3xdev** [comment]
Here is what I have in Settings > Colours And Fonts:

![image](https://user-images.githubusercontent.com/77379766/175427108-7a384307-7c1c-4011-939f-c288ede9fd29.png)

I have not changed anything else except Global Font & Global Theme on the styling side.

**Comment by eranif** [comment]
something is not right here, the font that you see in the preview window, is the font that should be used by output,build etc windows
can you click on the "Apply" button and see if it changes anything? (try changing themes and then click apply)

**Comment by python3xdev** [comment]
The problem is fixed.

![image](https://user-images.githubusercontent.com/77379766/175429067-9291393c-640b-4777-9b5d-c7969ab04897.png)

I used the 'Restore' button, then I changes the font and theme to the ones I wanted. But this did not work before. I have already tried this button at least twice. Not sure why it did not fix it self then. Third time's the charm.

Thank you for your help and time on this matter.

Daniel
